ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_search.py

```diff
@@ -295,3 +295,53 @@ def test_paddle_topk(
         sorted=sorted,
         test_values=False,
     )
+
+
+# @st.composite
+# def _broadcastable_trio(draw):
+#     dtype = draw(helpers.get_dtypes("valid", full=False))
+#     shapes_st = draw(
+#         hnp.mutually_broadcastable_shapes(num_shapes=3, min_dims=1, min_side=1)
+#     )
+#     cond_shape, x1_shape, x2_shape = shapes_st.input_shapes
+#     condition = draw(hnp.arrays(hnp.boolean_dtypes(), cond_shape))
+#     x = draw(helpers.array_values(dtype=dtype[0], shape=x1_shape))
+#     y = draw(helpers.array_values(dtype=dtype[0], shape=x2_shape))
+#     return condition, x, y, (dtype * 2)
```

Comment:
It's probably best not to commit commented-out code, if you're not going to use this then best to just delete it